addons/osfstorage/views.py

```diff
@@ -96,6 +96,27 @@ def osfstorage_update_metadata(payload, **kwargs):
 
     return {'status': 'success'}
 
+@must_be_signed
+@decorators.load_guid_as_target
+def osfstorage_get_storage_quota_status(target, **kwargs):
+    # Storage caps only restrict Nodes
+    if not isinstance(target, Node):
+        return {
+            'over_quota': False
+        }
+    # Storage calculation for the target has been accepted and will run asynchronously
```

Comment:
Quickfiles aren't calculated for storage and do not have storage caps. We'll need to bail out successfully around here if the node is a quickfiles node.

Reply:
Quickfiles should be handled by the `isinstance(target, Node)` because `QuickFilesNode` inherits from `AbstractNode` rather than `Node`. I verified this in the django shell.